conftest.py

```diff
@@ -324,6 +324,9 @@ def request_callback(request):
         rsps.add_callback(responses.GET, "http://example.com/file", callback=request_callback)
         rsps.add_callback(responses.GET, "https://example.com/file", callback=request_callback)
         rsps.add_callback(responses.GET, "http://example.com/file.ext?foo=bar", callback=request_callback)
+        rsps.add_callback(responses.HEAD, "http://example.com/file", callback=request_callback)
+        rsps.add_callback(responses.HEAD, "https://example.com/file", callback=request_callback)
+        rsps.add_callback(responses.HEAD, "http://example.com/file.ext?foo=bar", callback=request_callback)
```

Comment:
Is this required for this PR or is left from somewhere else?

Reply:
Weird that that ended up here. It's part of https://github.com/SwissDataScienceCenter/renku-python/pull/1516 I'll remove it here